app/components/NetworkStatus/NetworkStatus.tsx

```diff
@@ -42,7 +42,7 @@ const NetworkStatus = ({ status, error }: Props) => {
           <>
             <NetworkIndicator color={status?.isSynced ? smColors.green : smColors.orange} />
             <ProgressLabel>syncing</ProgressLabel>
-            <ProgressLabel>{getSyncLabelPercentage()}%</ProgressLabel>
+            <ProgressLabel>{progress}%</ProgressLabel>
```

Comment:
due to calculations and rounding "progress" can be > 100, in such case set progress label to 100 and send '100' as parameter to ProgressBar